crates/revm_precompiles/src/secp256k1.rs

```diff
@@ -1,9 +1,7 @@
+#[cfg(any(feature = "k256_ecrecover", feature = "secp256k1"))]
```

Comment:
Missing feature `cfg`, as this only works when either the `k256_ecrecover` or `secp256k1` feature flags are on. Ensures that `cargo t --no-default-features` succeeds.